.github/workflows/release-docker.yml

```diff
@@ -10,6 +10,9 @@ jobs:
     name: Docker build and push
     runs-on: ubuntu-latest
     steps:
+      - name: Check out the repository
```

Comment:
make sure it runs only before the step related to readme and use the latest checkout action

Reply:
done please check.

Reply:
now you do checkout twice, and still with old version of the action